### PR TITLE
#168131428 Users should be able to comment on Travel requests

### PR DESCRIPTION
**Pull request by drayzii**
#### What does this PR do?
Allows to comment on Travel requests
#### Description of Task to be completed?
- Implement adding comments on requests by their owner or manager
- Implement updating comments on requests by their owners
- Implement retrieving comments on requests by their owners or manager
#### How should this be manually tested?
- Clone this repo
- Checkout to branch `ft-add-requests-comments-168131428`
- Run `npm test`
- Run the application with `npm run start:dev` and after you successfully log in make these requests
1. `POST` request to `api/v1/requests/{request id}/comment` (Screenshot 1)
The payload should look like this
```
{
	"comment": "Comment Sample"
}
```
2. `PUT` request to `api/v1/requests/comments/{comment id}` (Screenshot 2)
The payload should look like this
```
{
	"comment": "Updated Comment"
}
```
3. `GET` request to `api/v1/requests/{request id}/comments` (Screenshot 3)

- After you make the request you should see a response that looks like the one in screenshots

- Alternatively, you can test using the Heroku Review App `https://mervels-bn-backend-stagi-pr-33.herokuapp.com`
#### Any background context you want to provide?
You can comment only on your requests unless you are logged in as a manager
#### What are the relevant pivotal tracker stories?
[#168131428](https://www.pivotaltracker.com/story/show/168131428)
#### Screenshots (if appropriate)
Screenshot 1
image

Screenshot 2
image

Screenshot 3
image
#### Questions:
N/A

### CODE REVIEW

File: src/middlewares/userRoles.js [hunk 1]
```diff
@@ -34,5 +36,38 @@ class Access {
...
   }
+
+  /**
+   * Checks if the user is the request owner or manager.
+   *@param {string} req  data.
+   * @param {string} res  data.
+   * @param {string} next data.
+   * @returns {string} object.
+   */
+  static async isRequestOwnerOrManager(req, res, next) {
```
Comment (willywax) [inline src/middlewares/userRoles.js]: function name too long

**Review by willywax — APPROVED** [review]
Looks good

**Review by Bahatiroben — APPROVED** [review]
LOOKS GOOD.

**Review by Paulvitalis200 — APPROVED** [review]
Good work on this feature

**Pull request merged**